cli/src/cmd/forge/script.rs

```diff
@@ -79,6 +78,18 @@ pub struct ScriptArgs {
 
     #[clap(flatten, next_help_heading = "EVM OPTIONS")]
     pub evm_opts: EvmArgs,
+
+    #[clap(
+        long,
+        help = "resumes previous transaction batch. DOES NOT simulate execution. respects nonce constraint "
```

Comment:
Can you clarify a bit? Without knowing much about `forge script`, I don't really know what this means.

Also for this + other flags we should start with a capital letter etc. like in the other commands (stylistic choice). I know you didn't write all the options here, but would you mind changing it? :smile: